_episodes/01-introduction.md

```diff
@@ -1,155 +1,127 @@
 
-![figure showing ESMValTool architecture]({{ page.root
-}}/fig/esmvaltool_architecture.png)
+- **Description** with relevant (citation) information
+- **Datasets** that should be analysed
+- **Preprocessor** steps that must be applied
+- **Diagnostic** scripts performing more specific evaluation steps
 
-> ## Discussion: (dis)advantages of this approach
->
-> Discuss or think about the pros and cons of this architecture for a moment.
-> Then unfold the box below to see our answers.
+An example recipe could look like this:
+
+```yaml
+documentation:
+  description: Example recipe
+  authors:
+    - lastname_firstname
+
+datasets:
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Amon, ensemble: r1i1p1, start_year: 1960, end_year: 2099}
+
+preprocessors:
+  global_mean:
+    area_statistics:
+      operator: mean
+
+diagnostics:
+  hockeystick_plot:
+    description: plot of global mean temperature change
+    variables:
+      temperature:
+        short_name: tas
+        preprocessor: global_mean
+    scripts: hockeystick.py
+```
+
+> ## Understanding the different section of the recipe
 >
+> Try to figure out the meaning of the different dataset keys. Hint: they can
+> be found in the documentation of ESMValTool.
 >
-> > ## See our thoughts
-> >
-> > - Streamlining common preprocessing steps ensures consistency in the
-> >   algorithms being used and the way they are executed. This facilitates
-> >   comparison and reproducibility.
-> > - Provenance and citation information can be tracked through the entire
-> >   workflow.
-> > - The core builds upon the [iris](https://scitools.org.uk/iris/docs/latest/)
-> >   package, which is quite strict in order to comply with CF conventions and
-> >   minimize unexpected results.
-> > - Recipes are easy to read and share.
-> > - A collection of recipes and diagnostic scripts is shipped with ESMValTool,
-> >   ready for re-use. Everyone can add to this collection.
-> > - The recipe format takes some getting used to and may be a bit less
-> >   flexible then working on the datasets directly.
-> > - Features or dataset support that are missing from ESMValCore can be a
-> >   limiting factor.
+> > ## Solution
+> > The keys are explained in the ESMValTool documentation, in the section `The
+> recipe format`, under
+> [datasets](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/overview.html#recipe-section-datasets)
 > {: .solution}
-{: .discussion}
+{: .challenge}
+
+## A library for reproducible climate science
 
-## Community
+More than a tool, ESMValTool is a library of publicly available recipes and
+diagnostic scripts. It comes with a large collection of recipes and diagnostic
+scripts to reproduce important results.
+
+> ## Explore the available recipes
+>
+> Go to the [documentation of esmvaltool](https://docs.esmvaltool.org/) and
+> explore the `available recipes` section. Which recipe(s) would you like to
+> try?
+{: .challenge}
+
```

Comment:
Perhaps we should also add a few example images here, include one or two easy to understand examples, and a "awesome OMG so many models" one, e.g. https://docs.esmvaltool.org/en/latest/_images/gleckler.png

Reply:
Not sure about that. Since we'll not be showing this page to the students. So I'd prefer to interactively browse to the documentation and then show the example output there.

Reply:
Fair enough :-)